utilities/pipelines/sharedScripts/Get-LocallyReferencedFileList.ps1

```diff
@@ -9,6 +9,9 @@ That means if module A references module B, which references module C, then all
 .PARAMETER FilePath
 Mandatory. The path to the template to investigate.
 
+.PARAMETER TemplateMap
+Optional. The hashtable of templatePath-templateContent to search in. Can be provided to speed up runtime.
+
 .EXAMPLE
 Get-LocallyReferencedFileList -FilePath 'C:/modules/key-vault/vault/main.bicep'
```

Comment:
can you please add an example of the new `TemplateMap` parameter?

Reply:
added